lib/github/components/Editor/index.js

```diff
@@ -76,18 +114,16 @@ class GithubEditor extends Component {
 
   revertHandler (e) {
     const { data: { repository }, view, path } = this.props
-    window.localStorage.removeItem(localStorageKey(repository, view, path))
+    window.localStorage.removeItem(getLocalStorageKey(repository, view, path))
```

Comment:
all interaction with `window.localStorage` can throw exceptions because of privacy or capacity issues. Probably want something like this: https://github.com/orbiting/cms-draft/blob/master/src/utils/localstorage.js